src/goodbase/goodbase_cli.py

```diff
@@ -594,11 +594,12 @@ def dependencies(binder: inject.Binder) -> None:
                     click.echo(click.style(f"\t{module}: {errmsg}", fg="yellow"))
         else:
             click.echo(click.style("No revision found", fg="red"))
-            click.echo(
-                "For more detailed information about what command is doing, use `--verbose` option\n"
-                "For more information about how to specify criteria, see"
-                " https://evergreen-ci.github.io/git-co-evg-base/concepts/criteria/",
-            )
+            if not verbose:
+                click.echo(
+                    "For more detailed information about what command is doing, use `--verbose` option\n"
+                    "For more information about how to specify criteria, see"
+                    " https://evergreen-ci.github.io/git-co-evg-base/concepts/criteria/",
```

Comment:
Do we also want to exclude this part or just the first part above?

Reply:
Good point. Moved it out of the if statement